src/components/Menu/MenuItem/__tests__/__snapshots__/menuItem-snapshots.jest.js.snap

```diff
@@ -3,20 +3,20 @@
 exports[`Snapshots renders correctly when disabled 1`] = `
 <li
   aria-current={false}
-  className="monday-style-menu-item monday-style-menu-item--disabled"
+  className="item disabled"
+  data-testid="menu-item_undefined-undefined"
```

Comment:
shouldn't that be avoided with default test ids?

Reply:
Fixed, check it out, please